src/ops/view.rs

```diff
@@ -272,6 +448,8 @@ fn print_deps(
             FeatureStatus::EnabledByUser | FeatureStatus::Enabled => enabled,
             FeatureStatus::Disabled => disabled,
         };
+
+        // TODO: Show the link.
```

Comment:
This TODO should be resolved

Reply:
Hi Ed~
I had resolved this on the next commit~

Please review again, thanks a lot😉~

Reply:
The result:
Only when the dependency version had reached out the `patch` level, it will be lead to the `crates.io/{its_name}/{its_version}` page, otherwise it wll be lead to the `crates.io/crates/{its_name}` page(Without exact version).

![image](https://github.com/hi-rustin/cargo-information/assets/24818903/cf387c58-bd31-4d0c-9258-36c3b6607a00)